microsoft365/outlook/calendar/pkg/commands/deleteEvent.go

```diff
@@ -29,9 +29,18 @@ func DeleteEvent(ctx context.Context, eventID, calendarID string, owner graph.Ow
 		return fmt.Errorf("failed to create client: %w", err)
 	}
 
-	if err := graph.DeleteEvent(ctx, c, trueEventID, trueCalendarID, owner); err != nil {
-		return fmt.Errorf("failed to delete event: %w", err)
+	if deleteSeries {
+		if err := graph.DeleteEventSeries(ctx, c, trueEventID, trueCalendarID, owner); err != nil {
+			return fmt.Errorf("failed to delete event series: %w", err)
+		} else {
+			fmt.Println("Event series deleted successfully")
+		}
```

Comment:
Same suggestion below.
```suggestion
		}
		fmt.Println("Event series deleted successfully")
```